src/note-components/archive/archive.jsx

```diff
@@ -1,19 +1,79 @@
 import "../notepage/note.css";
+import axios from "axios";
 import { Navbar } from "../navbar/navbar";
 import { Sidebar } from "../sidebar/sidebar";
+import { useNote} from "../../Context/note-context";
+import { useAuth } from "../../Context/authorization-context";
 
 const Archive = () =>{
-    return (
-        <div className="App">
-            <Navbar/>
-            <div className="main-container">
-                <Sidebar/>
-                <div className="right-cont">
-                    Archived Notes
+const { noteState, noteDispatch } = useNote();
+const { archive } = noteState;
+const { authState } = useAuth();
+const { token } = authState;
+
+const restoreNoteFromArchive = async (item) =>{
+console.log(item._id)
+try{
+    const response = await axios({
+        method:"post",
+        url:`/api/archives/restore/${item._id}`,
+        headers:{authorization:token}, 
+    });
+    console.log(response)
+    if(response.status === 200 || response.status === 201){
+        noteDispatch({type : "RESTORE_ARCHIVED_NOTE", payload : { archive : response.data.archives, note : response.data.notes}})
+    }
+}
+catch(error){
+    console.log(error);
+}
+}
+
+const moveToTrashFromArchive = async (item) => {
+console.log(item._id);
+try{
+const response = await axios({
+method:"delete",
+url:`/api/archives/delete/${item._id}`,
+headers:{authorization: token},
+});
+console.log(response)
+if(response.status === 200 || response.status === 201){
+noteDispatch({type : "DELETE_NOTE_FROM_ARCHIVE", payload : {archive : response.data.archives, trash : item}})
+}
+}
+catch (error){
+console.log(error);
+}
+}
```

Comment:
The spacing in the code lines should be consistent across the file. 

Use some code formatter to automate the task.

Reply:
Okay will do that. I actually formatted it as HTML, maybe that is why it is presented like this.